lib/libgcc_s/Makefile

```diff
@@ -54,6 +54,10 @@ SRCS+=		s_scalbnl.c
 .endif
 
 SYMBOL_MAPS+=	${.CURDIR}/Symbol-c18n.map
```

Comment:
This is wrong for non-c18n?

Reply:
What would the best way to check for this be? The rtld Makefile seems to check for purecap + aarch64. Is that sufficient?

Reply:
That sounds about right